feature-detects/mediastream.js

```diff
@@ -0,0 +1,13 @@
+define(['Modernizr'], function (Modernizr) {
+    Modernizr.addTest('mediarecorder', function(){
+        var isMediaRecorderSupported = false;
+
+        try {
+            MediaRecorder;
+            isMediaRecorderSupported = true;
+        } catch (err) {
+            console.log("no MediaRecorder");
```

Comment:
> Follow Patrick's advice about with the typeof, I leave this here in case you wanted some insight about funcions ;)

Pssh, here is where the false return should go 👀 (after the catch)
Extra clue: it is not necessary to declare `isMediaRecorderSupported`